styles.css

```diff
@@ -94,166 +189,236 @@ header {
 }
 
 .work-card {
+  box-sizing: border-box;
+  list-style: none;
   grid-column: 1;
   background: #fff;
   border-radius: 16px;
   border: 1px solid #dfe1e6;
-  width: 327px;
-  height: 552px;
+  width: 100%;
+  height: 100%;
+  min-height: 400px;
   padding: 16px;
+}
```

Comment:
- [ OPTIONAL] Great work making the changes requested by the first reviewer. The space is reduced when I view it on the standard desktop view screen. I would advise you to remove this CSS declaration from this element; `min-height: 400px`. I asked you to do so because when I view your cards on other desktop view screens, the space between the card and its border is much. I would strongly recommend you remove this declaration. 👍